proto/src/main/proto/pb.proto

```diff
@@ -1824,6 +1825,12 @@ message AutoConfirmSettings {
     string currency_code = 5;
 }
 
+message MoneroNodeSettings {
+    string blockchainPath = 1;
```

Comment:
These should use underscores in order to have correct camelcase in the typescript.